recipes/kangaru/all/conanfile.py

```diff
@@ -0,0 +1,41 @@
+from conans import ConanFile, tools, CMake
+import os, shutil
+
+class KangaruConan(ConanFile):
+    name = "kangaru"
+    description = "A dependency injection container for C++11, C++14 and later"
+    license = "MIT"
+    topics = ("conan", "gracicot", "kangaru", "DI", "IoC", "inversion of control")
+    homepage = "https://github.com/gracicot/kangaru/wiki"
+    url = "https://github.com/conan-io/conan-center-index"
+    options = {"reverse_destruction": [True, False],
+                "no_exception": [True, False]}
+    default_options = {"reverse_destruction": True,
+                        "no_exception": False}
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])

```

Comment:
```suggestion
        tools.get(**self.conan_data["sources"][self.version])
        os.rename(self.name + "-" + self.version, self._source_subfolder)

```

Reply:
I followed your suggestion here as well, but now I get an error with the conan center pre_export hook: "could not get source code". Could you help me with that one? I don't know what went wrong.

Reply:
Yes, let me check it. I'll send you a PR ASAP

Reply:
@AGreat1 it's all fine: https://gist.github.com/uilianries/1c5d4c73c0c1de95ff5c80384bde0955

Reply:
Then I'll look into my local environment later.